user_manual/files/access_webdav.rst

```diff
@@ -275,7 +275,7 @@ For example: ``net use Z: https://example.com/nextcloud/remote.php/dav/files/USE
 
 An alternative command syntax is::
 
-  net use Z: \\example.com@ssl\nextcloud\remote.php\dav /user:youruser
+  net use Z: \\example.com@ssl\DavWWWRoot\nextcloud\remote.php\dav\files\youruser /user:youruser
```

Comment:
That is if nextcloud is installed in a subfolder. For other reasons, this is not the preferred way, but it should be clear that the nextcloud folder is only required in this case.